testserv.c

```diff
@@ -202,18 +236,14 @@ readline(int fd)
 static void
 ckresp(int fd, char *exp)
 {
-    char *line;
-
-    line = readline(fd);
+    char *line = readline(fd);
```

Comment:
There are quite a few unrelated changes in this PR, such as the one on this line. What do you think about making those separately?

Reply:
I agree. I tend to add additional style improvements but i think it makes sense to not touch it here.